### ISSUE TITLE
2.0.0 release contains dependencies not compatible with PHP 7.4

### ISSUE THREAD

**Issue by Majkl578** [maglnet/ComposerRequireChecker#171]
Hi,
it seems the PHAR release uses libs incompatible with the recently released PHP 7.4.
I am getting a lot of errors coming from PhpParser:

```
Deprecated:  Invalid characters passed for attempted conversion, these have been ignored in phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Node/Scalar/String_.php on line 103
Stack trace:
  1. {main}() /usr/local/bin/composer-require-checker:0
  2. include() /usr/local/bin/composer-require-checker:10
  3. ComposerRequireChecker\Cli\Application->run() phar:///usr/local/bin/composer-require-checker/bin/composer-require-checker.php:32
  4. ComposerRequireChecker\Cli\Application->doRun() phar:///usr/local/bin/composer-require-checker/vendor/symfony/console/Application.php:145
  5. ComposerRequireChecker\Cli\Application->doRunCommand() phar:///usr/local/bin/composer-require-checker/vendor/symfony/console/Application.php:262
  6. ComposerRequireChecker\Cli\CheckCommand->run() phar:///usr/local/bin/composer-require-checker/vendor/symfony/console/Application.php:901
  7. ComposerRequireChecker\Cli\CheckCommand->execute() phar:///usr/local/bin/composer-require-checker/vendor/symfony/console/Command/Command.php:255
  8. ComposerRequireChecker\DefinedSymbolsLocator\LocateDefinedSymbolsFromASTRoots->__invoke() phar:///usr/local/bin/composer-require-checker/src/ComposerRequireChecker/Cli/CheckCommand.php:79
  9. ComposerRequireChecker\ASTLocator\LocateASTFromFiles->__invoke() phar:///usr/local/bin/composer-require-checker/src/ComposerRequireChecker/DefinedSymbolsLocator/LocateDefinedSymbolsFromASTRoots.php:27
 10. PhpParser\Parser\Multiple->parse() phar:///usr/local/bin/composer-require-checker/src/ComposerRequireChecker/ASTLocator/LocateASTFromFiles.php:35
 11. PhpParser\Parser\Multiple->tryParse() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Parser/Multiple.php:32
 12. PhpParser\Parser\Php7->parse() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Parser/Multiple.php:51
 13. PhpParser\Parser\Php7->doParse() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/ParserAbstract.php:158
 14. PhpParser\Parser\Php7->PhpParser\Parser\{closure:phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Parser/Php7.php:2173-2176}() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/ParserAbstract.php:284
 15. PhpParser\Node\Scalar\String_::parse() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Parser/Php7.php:2175
 16. PhpParser\Node\Scalar\String_::parseEscapeSequences() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Node/Scalar/String_.php:69
 17. preg_replace_callback() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Node/Scalar/String_.php:96
 18. PhpParser\Node\Scalar\String_::PhpParser\Node\Scalar\{closure:phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Node/Scalar/String_.php:97-109}() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Node/Scalar/String_.php:96
 19. hexdec() phar:///usr/local/bin/composer-require-checker/vendor/nikic/php-parser/lib/PhpParser/Node/Scalar/String_.php:103
```

**Comment by Ocramius** [comment]
I think the phar has very old deps in general: we could cut `2.1.0` to trigger a new phar build, @maglnet

**Comment by Majkl578** [comment]
> I think the phar has very old deps in general: 

I'd expect that since it's from March. :)

**Comment by maglnet** [comment]
Ack, I'll create a new release/phar soon.

**Comment by maglnet** [comment]
This should be fixed with release 2.1.0

@Majkl578 could you please verify?

**Comment by Majkl578** [comment]
Looks ok with PHP 7.4.1, thanks.
